src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/SelectSignatories.tsx

```diff
@@ -11,37 +11,30 @@ export const SelectSignatories = () => {
 
   const signatories = useUnit(signatoryModel.$signatories);
 
-  const onAddSignatoryClick = () => {
-    signatoryModel.events.addSignatory({ name: '', address: '', walletId: '' });
-  };
-
   return (
-    <div className="flex flex-1 flex-col">
-      <div className="flex flex-col gap-2">
+    <div className="flex flex-1 flex-col gap-y-4">
+      <div className="flex flex-col gap-y-2">
         {signatories.map((value, index) => (
           <Signatory
-            // TODO: Address and name maybe not unique by user input
-            key={value.address}
-            signatoryIndex={index}
+            key={`${value.name}_${value.address}_${index}`}
```

Comment:
Yeah, Alexander have recently changed it to `key={value.address}` because name+address can have duplicated
Seems that name+address can appear but less frequent than address, what is less likely to appear (index is unique in any case) name+address+index
@Asmadek @sokolova-an